specs/latest/1.0/index.html

```diff
@@ -877,6 +878,21 @@ <h4>Context creation parameters</h4>
               set to <code>false</code>, with the knowledge that a reduced-fidelity rendering mode
               should be used to improve performance.
             </dd>
+        <dt> <span class=prop-value>lowLatency</span></dt>
+            <dd>
+              <p>
+                If the value is true, the user agent will attempt to optimize the rendering of the
+                canvas to reduce latency, including potentially rendering from the front buffer.
+                This might desynchronize the canvas's display from the rest of the page and induce
```

Comment:
s/might desynchronsize/desynchromizes/
s/the canvas's display/Presentation of the Canvas/

Reply:
We would prefer to leave desynchronization of the canvas's presentation optional. It seems likely to us that some browser vendors would not do this, but instead might change their internal buffering strategy somehow.

Reply:
Note: in the current PR, have changed to use "presentation of the canvas".